tests/test_operation.py

```diff
@@ -31,214 +31,9 @@
 
 # pylint: disable=no-self-use, no-member, protected-access, pointless-statement
 
-# Operation subclasses to test
-op_classes = [getattr(qml.ops, cls) for cls in qml.ops.__all__]
-op_classes_cv = [getattr(qml.ops, cls) for cls in qml.ops._cv__all__]
-op_classes_gaussian = [cls for cls in op_classes_cv if cls.supports_heisenberg]
-op_classes_exception = {"PauliRot", "Projector"}
-
-op_classes_param_testable = op_classes.copy()
-for i in [getattr(qml.ops, cls) for cls in list(op_classes_exception)]:
-    op_classes_param_testable.remove(i)
-
-
-def U3(theta, phi, lam):
-    return Rphi(phi) @ Rphi(lam) @ Rot3(lam, theta, -lam)
-
-
-class TestOperation:
-    """Operation class tests."""
-
-    @pytest.mark.parametrize("test_class", op_classes_gaussian)
-    def test_heisenberg(self, test_class, tol):
```

Comment:
Sorry that I misused the PR to simplify and break up these huge tests!

I outsourced the CV tests to a new test class below.